### DISCUSSION TITLE
Domain change

### DISCUSSION THREAD

**Discussion by Tony-MF** [plausible/analytics#2856] [Self-Hosted Support] [answered]
Hi,

Is there any procedure that I should follow when I want to change my domain and don't lose any data?

I saw that you implemented a feature in cloud version, but I changed the domain today :)

**Comment by ruslandoga** [comment, marked as answer]
👋 @Tony-MF 

This should be possible in v2.0 pre-release: https://github.com/plausible/analytics/discussions/2896
